packages/ui/index.ts

```diff
@@ -80,6 +80,7 @@ export * from './src/composites/wui-select/index.js'
 export * from './src/layout/wui-flex/index.js'
 export * from './src/layout/wui-grid/index.js'
 export * from './src/layout/wui-separator/index.js'
+export * from './src/utils/JSXTypeUtil.js'
```

Comment:
This is to automatically inject elements into the dom without requiring to manually import the types util file everywhere that this needs to be used